CONTRIBUTING.md

```diff
@@ -1 +1,78 @@
 # Contributing to Chef Server
+
+Thank you for your interest in Chef Server!  We use **GitHub Issues**
+for issue tracking and contributions:
+
+1. Reporting an issue or making a feature request
+2. Submitting patches for new features or bug fixes
+
+## Submitting Patches
+
+All patches should be submitted as GitHub pull requests:
+
+1. Sign or be added to an existing
+[Contributor License Agreement (CLA)](https://supermarket.getchef.com/become-a-contributor).
+
+2. Create a GitHub Pull Request.
+
+3. If you are a Chef Software employee, submit an ad-hoc
+chef-server-12 build for your branch and post a link to in the pull
+request. For non Chef Software employees, your job will be submitted
+for you by one of the reviewers.
+
+4. Respond [Code Review](#cr) to code review comments to improve the
```

Comment:
This reads as "Respond Code Review to code review comments," probably not what you intended ;)